Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -25,7 +26,9 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                     return;
                 case UserState.NewBookingIsSelectedSelectingFloor:
                     UserStateStorage.userInfo[update.CallbackQuery.From.Id].MapId = Int32.Parse(update.CallbackQuery.Data);
-                    await new SendMonthCommand(mediator, telegraBotClient).SendAsync(update.CallbackQuery, "Please select start date month", "BACKNewBookingIsSelected");                    
+                    var officeNumber = await mediator.Send(new GetMapByIdQueryRequest() { Id = UserStateStorage.userInfo[update.CallbackQuery.From.Id].MapId });
```

Comment:
Make it this way
var officeNumber = await mediator.Send(new GetMapByIdQueryRequest() 
{ 
     Id = UserStateStorage.userInfo[update.CallbackQuery.From.Id].MapId 
});

Reply:
done